README.md

```diff
@@ -280,135 +325,104 @@ module.exports = {
 };
 ```
 
-In the example below we are setting the [`browserslist`](#browserslist) to target all the browsers with greater than `0.25%` market share but not IE 11. This information would then be passed to [Autoprefixer](https://github.com/postcss/autoprefixer) to add vendor prefixes and also to [Babel](https://babeljs.io/docs/en) so that it can transpile your Javascript which can be read by all the browsers you wish to support.
-
-### Polyfills
+In the example above we are setting the [`browserslist`](#browserslist) to target all the browsers with greater than `0.25%` market share but not IE 11. This information would then be passed to [Autoprefixer](https://github.com/postcss/autoprefixer) to add vendor prefixes and also to [Babel](https://babeljs.io/docs/en) so that it can transpile your Javascript which can be read by all the browsers you wish to support.
 
-## Deployment environments
+### Polyfilling of newer Javascript features
 
-### Using environment variables
+Underreact polyfills the following Javascript features
 
-Underreact allows you to inject environment variables **during the build time**. You can set them up by creating a `.env` file at the root of your project:
+- `Array.from`
+- `fetch`
+- `Object.assign`
+- `Promise`
+- `Symbol`
 
-```
-#.env
-SERVER=https://example.com
-```
+The above polyfills should allow you to freely use `for..of`, `async function() {}`, `fetch(..)` or `...{}` in your code.
 
-You can then use them in your client-side code:
+If your application needs any other polyfill, you can install it and import it at the top of your [`jsEntry`](#jsentry) file:
 
-```javascript
-const url = process.env.SERVER;
-fetch(`${url}/data`);
+```js
+// src/index.js
+import 'core-js/fn/set';
+import 'core-js/fn/map';
 ```
 
-All the variables in your environment **will not be automatically available** in your client-side code: any environment variable that you want to use in your app must be declared in the .env file. This is required to prevent unintentional leaking of env vars in your javascript bundle.
+### Using @babel/polyfill
```

Comment:
Do we need this section? Is there anything that is specific to Underreact?

Reply:
We can point them to Babel docs, but the information over there can be overwhelming. Hence I thought of putting the necessary information here. Do you think this otherwise?